3.0.0/README.md

```diff
@@ -0,0 +1,327 @@
+   1. [fillzoom](#39-fillzoom)
+   1. [grids](#310-grids)
+   1. [legend](#311-legend)
+   1. [maxzoom](#312-maxzoom)
+   1. [minzoom](#313-minzoom)
+   1. [name](#314-name)
+   1. [scheme](#315-scheme)
+   1. [template](#316-template)
+   1. [version](#317-version)
+1. [Examples](#4-examples)
+1. [Caching](#5-caching)
+
+# 1. Purpose
+
+This specification attempts to create a standard for representing metadata about multiple types of web-based map layers, to aid clients in configuration and browsing.
+
+# 2. File Format
+
+TileJSON manifest files use the JSON format as described in [RFC 8259](https://tools.ietf.org/html/rfc8259).
+
+# 3. Structure
+
+The following describes the structure of a TileJSON object. Implementations MUST treat unknown keys as if they weren't present. However, implementations MUST expose unknown key value pairs so users can optionally handle these keys. Implementations MUST treat invalid values for keys as if they weren’t present. If the the key is optional and the value is invalid, the default value MAY be applied. If the key is required, implementations MUST treat the entire TileJSON manifest file as invalid and refuse operation.
+
+*The word "implementation" in the following sections refers to services or tools that serve, generate, or validate TileJSON objects.*
+
+## 3.1 `tilejson`
+
+REQUIRED.
+
+A semver.org style version number as a string. Describes the version of the TileJSON spec that is implemented by this JSON object.
+
+```JSON
+{
+  "tilejson": "3.0.0"
+}
+```
+
+## 3.2 `tiles`
+
+REQUIRED.
+
+An array of tile endpoints. {z}, {x} and {y}, if present, are replaced with the corresponding integers. If multiple endpoints are specified, clients may use any combination of endpoints. All endpoint urls MUST be absolute. All endpoints MUST return the same content for the same URL. The array MUST contain at least one endpoint. The tile extension is NOT limited to any particular format. Some of the more popular are: mvt, vector.pbf, png, webp, and jpg.
+
+```JSON
+{
+  "tiles": [
+    "http://localhost:8888/admin/1.0.0/world-light,broadband/{z}/{x}/{y}.mvt"
+  ]
+}
+```
+
+## 3.3 `vector_layers`
+
+REQUIRED.
+
+A JSON object whose value is an array of JSON objects. Each of those JSON objects describes one layer of vector tile data. A `vector_layer` object MUST contain the `id` and `fields` keys, and MAY contain the `description`, `minzoom`, or `maxzoom` keys.
+
+#### 3.3.1 `id`
+
+REQUIRED.
+
+A string value representing the the layer id. For added context, this is referred to as the `name` of the layer in the [Mapbox Vector Tile spec](https://github.com/mapbox/vector-tile-spec/tree/master/2.1#41-layers).
+
+#### 3.3.2 `fields`
+
+REQUIRED.
+
+A JSON object whose keys and values are the names and descriptions of attributes available in this layer. Each value (description) MUST be a string that describes the underlying data. If no fields are present, the `fields` key MUST be an empty object.
```

Comment:
Should there be more specific recommendations for field's shape? An object with any number of keys, whose values are strings?

```js
fields: { [string]: string },
```

Reply:
Re-reading description and I think this is actually pretty well described already.